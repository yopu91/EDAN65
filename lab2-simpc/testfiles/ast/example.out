Program
  ParamFunc
    List
      Assignment
        IdUse
        IdUse
