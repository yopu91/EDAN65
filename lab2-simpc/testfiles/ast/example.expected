Program
  List
    ParamFunc
      IdUse
      List
        Assignment
          IdUse
          IdUse
